### PR TITLE
Fix wallet id display

### PR DESCRIPTION
**Pull request by sgliner-ledger**
Fix display error
Add tests parsing what is displayed on screen

### CODE REVIEW

File: workdir/app-near/src/get_wallet_id.c [hunk 1]
```diff
@@ -66,7 +66,7 @@ void handle_get_wallet_id(uint8_t p1, uint8_t p2, const uint8_t *input_buffer, u
 
     memcpy(tmp_ctx.address_context.public_key, public_key.W, 32);
 
-    snprintf(&wallet_id, sizeof(wallet_id), "%.H", 32, public_key.W);
+    snprintf(&wallet_id, sizeof(wallet_id), "%.*H", 32, public_key.W);
```
Comment (bigspider) [inline workdir/app-near/src/get_wallet_id.c:69]: I advocate getting rid of this custom format (easy to replace with a loop using `"%02x"` or `"%02X"`); but up to you, it's everywhere :/

File: workdir/app-near/tests/fixtures.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+# root of the repository
+repo_root_path: Path = Path(__file__).parent.parent
+
+# path of the folder of the currently running test
+
+ASSIGNMENT_RE = re.compile(
+    r'^\s*([a-zA-Z_][a-zA-Z_0-9]*)\s*=\s*(.*)$', re.MULTILINE)
+
+
+def get_app_version() -> str:
```
Comment (bigspider) [inline workdir/app-near/tests/fixtures.py]: Seems duplicated with the one in `conftest.py`?

**Review by bigspider — APPROVED** [review]
Very minor comments.

File: workdir/app-near/src/get_wallet_id.c (hunk 1 above)
Comment (sgliner-ledger) [reply workdir/app-near/src/get_wallet_id.c:69]: I used the loop before and @xchapron-ledger suggested this change :upside_down_face: 
Maybe we should try to enforce some sort of coding guidelines ?

File: workdir/app-near/tests/fixtures.py (hunk 2 above)
Comment (sgliner-ledger) [reply workdir/app-near/tests/fixtures.py]: I thought I removed this file ! 
Thanks :)

**Pull request merged**